api/models.py

```diff
@@ -75,3 +75,5 @@ class Node(ModelId):
     revision: Revision
     parent: Optional[PyObjectId]
     status: Optional[bool] = None
+    created: Optional[datetime] = None
```

Comment:
To set the created time automatically when the object is created:
```suggestion
    created: Optional[datetime] = Field(default_factory=datetime.utcnow)
```